Makefile

```diff
@@ -1,19 +1,36 @@
 #!/usr/bin/make
 
-NAME = "ProductOpener"
+ifeq ($(findstring cmd.exe,$(SHELL)),cmd.exe)
+    $(error "We do not suppport using cmd.exe on Windows, please run in a 'git bash' console")
+endif
+
+
+# use bash everywhere !
+SHELL := /bin/bash
+# some vars
 ENV_FILE ?= .env
+NAME = "ProductOpener"
 MOUNT_POINT ?= /mnt
 DOCKER_LOCAL_DATA ?= /srv/off/docker_data
-HOSTS=127.0.0.1 world.productopener.localhost fr.productopener.localhost static.productopener.localhost ssl-api.productopener.localhost fr-en.productopener.localhost
-COMPOSE_PROJECT_NAME ?= $(shell grep COMPOSE_PROJECT_NAME ${ENV_FILE}|cut -d '=' -f 2|sed -e 's/ //g')
+
 export DOCKER_BUILDKIT=1
 export COMPOSE_DOCKER_CLI_BUILD=1
 UID ?= $(shell id -u)
 export USER_UID:=${UID}
-
 export CPU_COUNT=$(shell nproc || echo 1)
 export MSYS_NO_PATHCONV=1
 
+# load env variables
+# also takes into account envrc (direnv file)
+ifneq (,$(wildcard ./${ENV_FILE}))
+    -include ${ENV_FILE}
+    -include .envrc
```

Comment:
I think it should never happen. If you have no .env, nothing will work.

Also it's about making things simple for most devs (including people running on windows). If we have specific problem in the future, we could change that.

Ideally the .env and .envrc should be change to have "?=" instead of "=" but I don't know how to do it in a simple way ! Note that even with "=", [`-e` option can make environment have precedence](https://www.gnu.org/software/make/manual/html_node/Environment.html)